server/server.py

```diff
@@ -20,6 +23,19 @@ def notify_game_created(challenge_id, game_id):
     )
 
 
+async def accept_challenge(data, client):
+    game_id = data.get('data', {}).get('game_id')
+    if game_id is None:
+        await notify_error_to_client(
+            client,
+            GameIdException,
```

Comment:
I think you should do `str(GameIdException)`

Reply:
done !